src/components/users/account/ProfileEditForm.tsx

```diff
@@ -71,13 +70,15 @@ export default function ProfileEditForm (): ReactElement {
     if (profile != null) {
       // Update the profile object in the state
       setProfile(profile)
-      // Set the flag that indicates the user just submitted a change
-      setJustSubmitted(true)
+      // Removed old state for Snackbar
```

Comment:
Can you remove this reference to the Snackbar component? I don't think it's useful to know it used to be here.